roles/cluster/cluster_deploy_kepler/defaults/main/config.yml

```diff
@@ -0,0 +1,4 @@
+# Auto-generated file, do not edit manually ... 
+# Toolbox generate command: repo generate_ansible_default_settings
+# Source component: Cluster.deploy_kepler
+
```

Comment:
this confirms that `repo generate_ansible_default_settings` GH action did not run, it should have caught this. Weird 🤔 (did you understand it didn't run?)

Reply:
Exactly I was getting the lint warning, so I had to render the job and make it explicit to fail. In the current version I removed the silent option and counted the lines, if the diff is empty then the parameters should be synchronized, otherwise the PR needs to be adjusted

Reply:
The action actually runs but unless you include the changes in the PR they wont be included. At least for me it was failing I usually don't restrict the branches, maybe the problem was that your branch for the PR wasnt main?

Reply:
> The action actually runs but unless you include the changes in the PR they wont be included.

I don't think so, that might be true for ansible-lint, which tries to be smart,
but `generate_ansible_default_settings` isn't smart (I wrote it :D) so it should regenerate all the `config.yaml` files, and fail if there is _any_ diff between the committed code and what has been regenerated 🤔